<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assessment scientific work</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QComboBox" name="typeScientificResearchComboBox">
     <item>
      <property name="text">
       <string>fundamental study</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>applied research</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>experimental development</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>military-scientific support</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>innovative research</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameStageLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name stage:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="stageLaborCostsScientificDoubleSpinBox">
     <property name="maximum">
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Stage labor costs (scientific result):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>General labor costs:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="generalLaborCostsDoubleSpinBox">
     <property name="maximum">
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="resultKNRLabel">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>true</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameWorkLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Type of scientific research:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Type of scientific results:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QDoubleSpinBox" name="stageLaborCostsPracticalDoubleSpinBox"/>
   </item>
   <item row="3" column="6" colspan="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" colspan="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name work:</string>
     </property>
     <property name="buddy">
      <cstring>nameWorkLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="10">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Quality indicators of scientific work</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="amountBorrowingDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="severityDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Objectness:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="noveltyDoubleSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>8.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="objectivityDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Evidence and substantiation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Novelty:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="evidenceSubstantiationComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="evidenceSubstantiationDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Severity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Objectivity:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="noveltyComboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="severityComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="objectivityComboBox"/>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="systemDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>System:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Verifiability:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="verifiabilityComboBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="systemComboBox"/>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="verifiabilityDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="objectnessDoubleSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="objectnessComboBox"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Amount of borrowing:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="10">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name result</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Criterion for assessing the quality</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value of the scientific result</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Labor expenditures (scientific result)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Labor expenditures (practical result)</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addResultButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add result</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Сriterion for assessing the quality:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Stage labor costs (practical result):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="typeScientificResultComboBox">
     <item>
      <property name="text">
       <string>methodical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>theoretical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>synergetic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>innovative</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>systemic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2" colspan="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Level of scientific results:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="levelScientificResultComboBox">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
